### PR TITLE
Added navbar element for Drivers list and placeholder for Drivers Page @ /drivers

### LINKED ISSUE #56: Add a navbar element for Driver List as a top level button (for now) that routes to a page /drivers
# Depends on

At least the first two stories from Epic #62 (any user can add their cell phone) must be completed first; i.e the ones that add cell phone to the entity, and add a backend route for updating the cell phone.

# User Story

* As a rider, admin, or driver
* I can see a list of all drivers, along with their cell phone numbers
* So that I can contact drivers as needed

# Acceptance Criteria

- [ ] Add a navbar element for Driver List as a top level button (for now) that routes to a page `/drivers`, which could be a placeholder at first (i.e. a DriverListPage component.)

### PR DESCRIPTION
**Pull request by Aasish-Virjala**
In this PR, I added a navbar element for the Drivers list page at /drivers. Added routes to App.js to serve a placeholder page for admins, drivers, and riders. + Tests.

Screenshot 2023-08-25 at 6 16 37 AM

Closes #56

### CODE REVIEW

**Review by pabucsb — DISMISSED** [review]
LGTM

**Review by pabucsb — DISMISSED** [review]
LGTM

File: frontend/src/main/components/Nav/AppNavbar.js [hunk 1]
```diff
@@ -87,6 +87,11 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                   </NavDropdown>
                 )
               }
+              {
+                isParticipant(currentUser) && (
+                  <Nav.Link data-testid="appnavbar-driver-link" href="/drivers">Drivers</Nav.Link>
```
Comment (Aasish-Virjala) [inline frontend/src/main/components/Nav/AppNavbar.js:92]:
```suggestion
                  <Nav.Link data-testid="appnavbar-driver-link" href="/drivers">Drivers List</Nav.Link>
```

**Review by pabucsb — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
There are some things I would have liked to have seen changed, but I'll let them slide.  

Good enough to merge.

**Pull request merged**